DEVELOPER.md

```diff
@@ -14,10 +14,12 @@ See http://kafka.apache.org/documentation.html#consumerconfigs for details about
 
     input {
         kafka {
-            topic_id => ... # string (required), The topic to consume messages from
+            topic_id => ... # string (required), The topic to consume messages from. Can be a java regular expression for whitelist of topics.
```

Comment:
topic_match or something would seem more appropriate, but I agree regarding backwards compat. We can gradually deprecate `topic_id` if there is consensus that another name is more correct and provide backwards compat.